.prettierignore

```diff
@@ -1 +1,4 @@
-**
\ No newline at end of file
+# Ignore artifacts:
+build
+coverage
+dist
```

Comment:
is contractData generated? if so we could add to here

Reply:
it's generated when the contracts are deployed to different networks so it's actually important to keep the ABI and the deployed network information. Admittedly there's a lot that gets generated we never use so it's a little frustrating but rather than write something custom to prune easiest is to keep as is.

Reply:
sounds good! just add to this file if you want to exclude generated stuff from prettier reformatting

Reply:
ahhh thought i was lookin at gitignore. love it. muah